kubernetes-elasticsearch-cluster/create-cluster.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+help_text="Usage: kubernetes-elasticsearch-cluster/create-cluster.sh <dataset> <zone>
+  where <dataset> is the name of a directory in dataset_config/
+  <zone> is only needed if 'gcloud app create' hasn't yet been run in this project.
+Run this script from project root"
+
+if (( $# != 1 && $# != 2 )); then
+  echo "$help_text"
+  exit 1
+fi
+
+dataset=$1
+zone=""
+project_id=$(jq --raw-output '.project_id' dataset_config/${dataset}/deploy.json)
+gcloud config set project $project_id
+
+# Use the zone argument if present, otherwise try to lookup the App Engine location 
+# for this project.
+if (( $# == 2 )); then
+  zone=$2
+else
+  describe_text=$(gcloud app describe) || true
+  app_engine_region=$(echo "$describe_text" | grep locationId | awk '{print $2}')
+  # If no zone was found in App Engine, print the help text describing that it must
+  # be passed in as an argument to this script.
+  if [ -z "$app_engine_region" ]; then
+    echo "$help_text"
+    exit 1
+  fi
+
+  # Locations with only a single region (e.g. "us-central1") do not have the last 
+  # number returned in 'gcloud app describe' location (only "us-central"). 
+  # Check if the location is missing a digit as its last character and, if so,
+  # add a "1".
+  last_char=${app_engine_region: -1}
+  if [[ ! $last_char =~ ^[0-9]+$ ]]; then
+  	app_engine_region+="1"
+  fi
+  zone="${app_engine_region}-a"
+fi
+
+data_node_machine_type=$(jq --raw-output '.data_node_machine_type' dataset_config/${dataset}/deploy.json)
```

Comment:
Let's have this file only create the cluster. Move creating data node pool, which is ES-related, to deploy.sh

Reply:
Per offline discussion, I think it makes more sense to leave in this script.